src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -441,5 +435,42 @@ internal void SafeOperation(Action op)
                 logger.WriteLine($"Daemon error: {ex.ToString()}");
             }
         }
+
+        #region IDisposable Support
```

Comment:
The Dispose pattern was implemented correctly before because the class was `sealed`.
However, it's no longer sealed to make it more testable, so we now need a `protected virtual Dispose` method.

This is the standard pattern implementation applied by VS.